builtin/providers/kubernetes/validators.go

```diff
@@ -80,3 +92,70 @@ func validateResourceList(value interface{}, key string) (ws []string, es []erro
 	}
 	return
 }
+
+func validateActiveDeadlineSeconds(value interface{}, key string) (ws []string, es []error) {
+	v := value.(int)
+	if v <= 0 {
+		es = append(es, fmt.Errorf("%s must be greater than 0", key))
+	}
+	return
+}
+
+func validateProbeFailureThreshold(value interface{}, key string) (ws []string, es []error) {
+	v := value.(int)
+	if v <= 0 {
+		es = append(es, fmt.Errorf("%s must be greater than 0", key))
+	}
+	return
+}
+
+func validateProbePeriodSeconds(value interface{}, key string) (ws []string, es []error) {
+	v := value.(int)
+	if v <= 0 {
+		es = append(es, fmt.Errorf("%s must be greater than 0", key))
+	}
+	return
+}
+
+func validateProbeSuccessThreshold(value interface{}, key string) (ws []string, es []error) {
+	v := value.(int)
+	if v <= 0 {
+		es = append(es, fmt.Errorf("%s must be greater than 0", key))
+	}
+	return
+}
+
+func validateProbeTimeoutSeconds(value interface{}, key string) (ws []string, es []error) {
+	v := value.(int)
+	if v <= 0 {
+		es = append(es, fmt.Errorf("%s must be greater than 0", key))
+	}
+	return
+}
```

Comment:
Since they're all the same, how would you feel about merging all the above validation functions into one and call it e.g. `validatePositiveInteger`?